src/main/java/ledger/database/entity/IgnoredExpression.java

```diff
@@ -0,0 +1,44 @@
+package ledger.database.entity;
+
+/**
+ * Created by gert on 2/7/17.
+ */
+public class IgnoredExpression implements IEntity {
+    private int expressionId;
+    private String expression;
+    private Boolean matchOrContain;
```

Comment:
Unclear variable name. True of false doesn't tell you which setting is being used